Src/AutoFoq/AutoFoqCustomization.fs

```diff
@@ -18,7 +18,8 @@ type AutoFoqCustomization(relay: ISpecimenBuilder) =
         AutoFoqCustomization(
             FilteringSpecimenBuilder(
                 MethodInvoker(
-                    FoqMethodQuery()),
+                    FoqMethodQuery(
+                        Fixture())),

```

Comment:
I think we need to find a solution to this. Having *two* Fixture instances in play is bound to blow up in our faces sooner or later.